src/libraries/System.Net.NetworkInformation/ref/System.Net.NetworkInformation.cs

```diff
@@ -138,6 +138,8 @@ public abstract partial class IPGlobalProperties
         public abstract System.Net.IPEndPoint[] GetActiveUdpListeners();
         public abstract System.Net.NetworkInformation.IcmpV4Statistics GetIcmpV4Statistics();
         public abstract System.Net.NetworkInformation.IcmpV6Statistics GetIcmpV6Statistics();
+        [System.Runtime.Versioning.UnsupportedOSPlatform("illumos")]
+        [System.Runtime.Versioning.UnsupportedOSPlatform("solaris")]
```

Comment:
I'm not sure if we need to care about unsupported OSes. 
There is really now way how somebody can target them right now AFAIK.

Reply:
We have them as targets for this project:
https://github.com/dotnet/runtime/blob/8e92a6fe4d89934eba274e2a247c800a725d6de9/src/libraries/System.Net.NetworkInformation/src/System.Net.NetworkInformation.csproj#L4
I assume that's the reason why it's firing the warnings. 

@buyaa-n can probably share more detail about the inner working of the analyzer.

Reply:
> I assume that's the reason why it's firing the warnings.

Right when there is a target platform in the TFM that build going to be for that platform for the analyzer. Sounds like the case the API is supported on certain platforms and throws on all others, if you cannot use SupportedOSPlatform attributes with the realistic platforms then we can skip the warnings for these unrealistic platforms.

The PNSE analyzer will not be added to the repo its just one time run  to detect missing annotations